package.json

```diff
@@ -123,6 +123,49 @@
         "command": "io.orta.jest.coverage.toggle",
         "title": "Jest: Toggle Coverage Overlay"
       }
+    ],
+    "debuggers": [
+      {
+        "type": "node",
+        "label": "Debug Jest tests using vscode-jest"
+      },
+      {
+        "type": "vscode-jest-tests",
+        "label": "Debug Jest tests using vscode-jest",
+        "languages": ["javascript", "javascriptreact", "typescript", "typescriptreact"],
```

Comment:
I would prefer to target only `*.test.(j|t)sx?` files, but when creating a separate language definition, it seems like I can't base it on the javascript(react)/typescript(react) definitions (including syntax highlighting, IntelliSense, ...).

Reply:
I think that should be fine for now, if it's a platform limitation...